apps/mark-scan/backend/src/server.test.ts

```diff
@@ -1,25 +1,102 @@
-import { fakeLogger } from '@votingworks/logging';
+import { LogEventId, fakeLogger } from '@votingworks/logging';
 import tmp from 'tmp';
 import { buildMockInsertedSmartCardAuth } from '@votingworks/auth';
+import {
+  BooleanEnvironmentVariableName,
+  getFeatureFlagMock,
+} from '@votingworks/utils';
+import { MockPaperHandlerDriver } from '@votingworks/custom-paper-handler';
+// import { sleep } from '@votingworks/basics';
 import { PORT } from './globals';
-import { start } from './server';
+import { resolveDriver, start } from './server';
 import { createWorkspace } from './util/workspace';
-import { getMockStateMachine } from '../test/app_helpers';
+
+const featureFlagMock = getFeatureFlagMock();
+jest.mock('@votingworks/utils', (): typeof import('@votingworks/utils') => {
+  return {
+    ...jest.requireActual('@votingworks/utils'),
+    isFeatureFlagEnabled: (flag: BooleanEnvironmentVariableName) =>
+      featureFlagMock.isEnabled(flag),
+  };
+});
+
+beforeEach(() => {});
```

Comment:
Cleaned up in https://github.com/votingworks/vxsuite/pull/4628/commits/1fed52f30b1b41daae362e59af77ba38c864bcfc